### PR TITLE
(PC-17681) fix(cookies): Opt out by default from firebase and amplitude

### PR DESCRIPTION
**Pull request by aliraiki**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-17681

## Checklist

I have:

- [ ] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [ ] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|     |         |                 |                  |                  |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/cookies/helpers/startTrackingAcceptedCookies.test.ts [hunk 1]
```diff
@@ -1,24 +1,15 @@
...
-const mockDisableAmplitudeCollection = jest.fn()
-const mockEnableAmplitudeCollection = jest.fn()
-
-jest.mock('libs/amplitude', () => ({
-  amplitude: () => ({
-    disableCollection: mockDisableAmplitudeCollection,
-    enableCollection: mockEnableAmplitudeCollection,
-  }),
-}))
-
```
Comment (Lucasbeneston) [inline src/features/cookies/helpers/startTrackingAcceptedCookies.test.ts:16]: 👍

**Review by Lucasbeneston — APPROVED** [review]
Plus propre comme ça !

**Comment by Lucasbeneston** [comment]
Dommage pour le tests d'Amplitude :(

**Comment by aliraiki** [comment]
J'ai essayé mais je trouve pas de moyen pour tester le fait que ça opt out par défaut

**Pull request merged**